tests/conftest.py

```diff
@@ -285,7 +296,9 @@ def updated_dsc_component_state(
 
 
 @pytest.fixture(scope="package")
-def enabled_modelmesh_in_dsc(dsc_resource: DataScienceCluster) -> Generator[DataScienceCluster, Any, Any]:
+def enabled_modelmesh_in_dsc(
+    dsc_resource: DataScienceCluster,
```

Comment:
You are updating a resource in a packaged scope fixture using session scoped dsc. Is this not causing side effects in teardown?